dashboard/lib/tasks/asset_sync.rake

```diff
@@ -0,0 +1,11 @@
+namespace :assets do
+  desc 'Synchronize assets to S3'
+  task :sync => :environment do
+    # AWS CLI implements an optimized `sync` utility without any Ruby SDK equivalent.
+    `aws s3 sync #{CDO.dashboard_dir}/public/assets s3://#{CDO.assets_bucket}/#{rack_env}/assets --acl public-read --cache-control 'max-age=31536000'`
```

Comment:
:+1: sounds correct!

As a future optimization, it would be possible to use the `--delete` flag for the sync along with invoking [`assets:clean`](https://github.com/rails/sprockets-ails/blob/master/README.md#usage) (which retains the most recent 3 copies of precompiled assets) before (or enhancing) `assets:precompile`.

I'd consider this a premature optimization until our S3 storage costs start to become a concern, though. `production-daemon`'s current `public/assets` folder is currently `1.6GB` which costs about 5 cents per month on S3, so I'd say we have a bit of runway there.